cmd/controller/main.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2020 TriggerMesh, Inc
```

Comment:
Does the placement of the comma make sense? just curious

Reply:
Not sure, I copied the existing boilerplate from other files. @sebgoa?

Reply:
actually, should be "TriggerMesh Inc."